src/module.model/_model.probes.ts

```diff
@@ -59,6 +62,14 @@ export class ModelProbeIndicator extends ProbeIndicator {
       return this.withDead('model', 'synced blocks are more than 2 blocks behind', details)
     }
 
+    if (now() - highest.time >= 90 * 60 && index >= defid) {
```

Comment:
```suggestion
    if (index === defid && now() - highest.time >= 90 * 60) {
```

Since index defid can experience a rollback, let's make it such that index is `always === to defid` to make sure the stale condition is only checked if `Model === DeFid`.